libpod/container_internal.go

```diff
@@ -150,7 +150,8 @@ func (c *Container) syncContainer() error {
 	// If runtime knows about the container, update its status in runtime
 	// And then save back to disk
 	if (c.state.State != ContainerStateUnknown) &&
```

Comment:
Might be good to create a "isContainerReady()"  type of function at some point so if we need to adjust down the line we can just change the one function.  Might be a little more readable too.  Not a stopper for this PR, just a suggestion.

Reply:
I was thinking the same thing, since we have this multi line if in many places.